src/embed/conversation.ts

```diff
@@ -229,7 +229,13 @@ export class SpotterEmbed extends TsEmbed {
         } = this.viewConfig;
 
         if (!worksheetId) {
-            this.handleError(ERROR_MESSAGE.SPOTTER_EMBED_WORKSHEED_ID_NOT_FOUND);
+            this.handleError(ERROR_MESSAGE.SPOTTER_EMBED_WORKSHEED_ID_NOT_FOUND, {
+                errorType: ErrorDetailsTypes.VALIDATION_ERROR,
```

Comment:
why is errorType and code both needed ? what is the use for source here ?
from customer's point of view arent all errors from sdk